packages/page-signing/src/Verify.tsx

```diff
@@ -56,13 +59,26 @@ function Verify ({ className = '' }: Props): React.ReactElement {
         if (isValidSr) {
           cryptoType = 'sr25519';
         } else {
-          isValid = false;
+          let verification;
```

Comment:
Should probably just use `signatureVerify` here and get rid of the specifics for sr/ed. We get the valid & type for free.

(Should actually have been done before, as it stands it is a duplication of what we have in util-crypto)